src/gui/folder.cpp

```diff
@@ -81,7 +81,9 @@ auto priorityC()
     return QStringLiteral("priority");
 }
 
-constexpr int SettingsVersionC = 5;
+// We don't use these versions anymore, see https://github.com/owncloud/client/issues/10473 . This
+// value is only written, and that prevents older clients from loading these newer settings.
+constexpr int SettingsVersionC = 13;
```

Comment:
Hmmm could it make sense to move all those 13 to ConfigFile::UnusedLegacySettingsVersionNumber ?

Reply:
done